.expeditor/config.yml

```diff
@@ -1,4 +1,4 @@
-# Documentation available at http://expeditor-docs.es.chef.io/
+# Documentation available at https://expeditor-docs.es.chef.io/
```

Comment:
there isn't actually an https version of this site.

Reply:
This should be https://expeditor.chef.io